cpp/src/parquet/arrow/arrow_reader_writer_test.cc

```diff
@@ -4092,7 +4092,7 @@ TEST_P(TestArrowWriteDictionary, Statistics) {
       {{"b", "a"}, {"b", "a"}}, {{"c", "c"}, {"c", "c"}}, {{"d", "a"}, {"d", "a"}}};
   const std::vector<std::vector<std::vector<bool>>> expected_has_min_max_by_page = {
       {{true, true}, {true, true}},
-      {{true, true}, {true, true}},
+      {{true, false}, {true, false}},
```

Comment:
Why did this change?

Reply:
https://github.com/apache/arrow/pull/35242#issuecomment-1517491968

This is mentioned here. Let me explain it:
1. Previously, `PageHeader` reuse same object, so `__isset` would not been cleared
2. So, `has_min_max` will keep "true" if first page has min-max, and second page not has.
3. In test case, the second page should not has min-max, but it has been set. I think previous test is a bug here. You can check it again

Reply:
That makes sense. Thanks for explaining.